sdks/java/io/google-cloud-platform/pom.xml

```diff
@@ -404,4 +416,15 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.api</groupId>
```

Comment:
Are you using shadow jar now, can this be removed?

Reply:
This code is not using the shadow jar. I'll have to make that change after moving (copying) this code over to the repo where the shadow jar is produced.

Chamikara, Eugene -- the long-term plan is not to maintain two copies of this code; however, until our client bits can be released to open source, we're planning to build this code from a separate repo so that we can keep the produced artifacts in sync with our client bits.